src/sb_gazebo/launch/gazebo_open_world.launch

```diff
@@ -0,0 +1,57 @@
+<launch>
+  <!--
+    Launch File For Opening Worlds - From the ros_gazebo package
+    
+    Snowbots Modifications:
+        Added optional `tf` argument so we can remap the tf topic for gazebo to prevent tf tree conflicts
```

Comment:
This is the main reason I added this. We wanted to be able to generate and use the TF tree based solely off our localisation system, but a bunch of the plugins that we use in Gazebo publish TF's that interefere, so we solve that problem by giving them their own TF tree.